neuraxle/steps/features.py

```diff
@@ -0,0 +1,88 @@
+"""
+Feature Steps
+==========================================================
+You can find here steps that featurize your data.
+
+..
+    Copyright 2019, Neuraxio Inc.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+"""
+from neuraxle.base import NonFittableMixin, BaseStep
+from neuraxle.pipeline import Pipeline
+from neuraxle.steps.flow import ChooseOneOrManyStepsOf
+from neuraxle.steps.numpy import NumpyFFT, NumpyAbs, NumpyFlattenDatum, NumpyConcatenateInnerFeatures, NumpyMean, \
+    NumpyMedian, NumpyMin, NumpyMax, NumpyArgMax
+from neuraxle.union import FeatureUnion
+
+
+class FFTPeakBinWithValue(FeatureUnion):
+    """
+    Compute peak fft bins (int), and their magnitudes' value (float), to concatenate them.
+
+    .. seealso::
+        :class:`~neuraxle.base.BaseStep`,
+        :class:`~neuraxle.base.NonFittableMixin`
```

Comment:
```suggestion
        :class:`~neuraxle.base.NonFittableMixin`
        :class:`~neuraxle.steps.numpy.NumpyFFT`
```
and also Cheap3DTo2DTransformer